core/src/subgraph/instance_manager.rs

```diff
@@ -206,48 +205,27 @@ impl<S: SubgraphStore> SubgraphInstanceManager<S> {
 
     pub async fn hashes_to_read_store<C: Blockchain>(
```

Comment:
Maybe this should be renamed now that it returns `SourceableStore` not `ReadStore`